lib/ansible/modules/cloud/openstack/os_ironic.py

```diff
@@ -102,6 +102,14 @@
           description:
             - size of first storage device in this machine (typically /dev/sda), in GB
           default: 1
+        capabilities:
+          description:
+            - special capabilities for the node, such as boot_option, node_role etc
+          default: ""
+        root_device:
+          description:
+            - Root disk selections. eg: /dev/sda
```

Comment:
Having an extra `: ` confuses the yaml parser, perhaps
`Root disk selection, such as C(/dev/sda)`

The example and documentation don't seem to match up. In the `EXAMPLE` it looks like it may support a list